models/DataObject/Data/ObjectMetadata/Dao.php

```diff
@@ -118,6 +118,7 @@ public function createOrUpdateTable(DataObject\ClassDefinition $class)
         $table = 'object_metadata_' . $classId;
 
         $this->db->executeQuery('CREATE TABLE IF NOT EXISTS `' . $table . "` (
+              `id` int(11) UNSIGNED NOT NULL AUTO_INCREMENT,
```

Comment:
@robertSt7 this conflicts with changes done on 11.0 https://github.com/pimcore/pimcore/commit/4d93f1f93eb8a5544ea4ae8c4345d5fec181b0f3#diff-6c6cbcb57ed8138d7a968f573cd8bc8a3aeaef3c76bd0ed634a53d2822909712R88

please see if a follow up is required on 11.0. thanks!